src/VisualStudio/Core/Test.Next/Remote/SerializationValidator.cs

```diff
@@ -78,9 +78,16 @@ public SerializationValidator(HostWorkspaceServices services)
             Services = services;
         }
 
-        public async Task<T> GetValueAsync<T>(Checksum checksum)
+        private async Task<SolutionAsset> GetRequiredAssetAsync(Checksum checksum)
         {
             var data = await AssetStorage.GetTestAccessor().GetRequiredAssetAsync(checksum, CancellationToken.None).ConfigureAwait(false);
+            Contract.ThrowIfNull(data);
+            return new(checksum, data);
+        }
+
+        public async Task<T> GetValueAsync<T>(Checksum checksum)
+        {
+            var data = await GetRequiredAssetAsync(checksum).ConfigureAwait(false);
```

Comment:
this is all test code, which expects to always find these values.  so it's ok for it to put in these contract calls and requirements.